src/proto/CMakeLists.txt

```diff
@@ -7,6 +7,10 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
     list(APPEND CMAKE_PREFIX_PATH "${Protobuf_PREFIX_PATH}")
 endif()
 find_package(Protobuf REQUIRED)
+if (NOT "${Protobuf_VERSION}" VERSION_LESS 4.22)
```

Comment:
How about changing to: if ("${Protobuf_VERSION}" VERSION_GREATER_EQUAL 4.22)
This seems more straight-forward.